src/geanyobject.c

```diff
@@ -32,6 +32,10 @@
  * Emitted just after loading main keyfile settings.
  */
 
+/**
+ * @file geanyobject.h
+ * Definition of the global GeanyObject */
+
```

Comment:
> Why? ...

Because it's not useful to normal plugins and only used internally by the bindings generated by a core script, so there's no point in having it clutter up the normal API documentation. I thought that's what `@gironly` is for. No big deal though.